### PR TITLE
Add deprecation warning

### PR DESCRIPTION
**Pull request by DanielHougaard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,24 +7,28 @@
...
-  - [Get Secrets](#get-secrets)
-  - [Get Secret](#get-secret)
-  - [Create Secret](#create-secret)
-  - [Update Secret](#update-secret)
-  - [Delete Secret](#delete-secret)
-- [Cryptography](#cryptography)
-  - [Create Symmetric Key](#create-symmetric-key)
-  - [Encrypt Symmetric](#encrypt-symmetric)
-  - [Decrypt Symmetric](#decrypt-symmetric)
+-   [Links](#links)
```
Comment (maidul98) [inline README.md:16]: It looks like your editor added lots of extra spaces. Is it possible you can remove these so the pr reflects only the changes you made?

**Review by maidul98 — COMMENTED** [review]
Otherwise everything else looks good

File: README.md (hunk 1 above)
Comment (DanielHougaard) [reply README.md:16]: Yep. I spoke to Tony about this prior to the PR, and we didn't seem to have a standard for formatting, which has led to a lot of extra spaces

Comment (DanielHougaard) [reply README.md:16]: Yep. I spoke to Tony about this prior to the PR, and we didn't seem to have a standard for formatting, which has led to a lot of extra spaces

**Review by maidul98 — APPROVED** [review]

**Pull request merged**